services/director-v2/tests/unit/with_dbs/test_modules_dynamic_sidecar_docker_api.py

```diff
@@ -448,24 +445,6 @@ async def test_create_service(
     assert service_id
 
 
-async def test_inspect_service(
-    service_spec: dict[str, Any],
-    cleanup_test_service_name: None,
-    docker_swarm: None,
-):
-    service_id = await docker_api.create_service_and_get_id(service_spec)
-    assert service_id
-
-    service_inspect = await docker_api.inspect_service(service_id)
-
-    assert service_inspect["Spec"]["Labels"] == service_spec["labels"]
-    assert service_inspect["Spec"]["Name"] == service_spec["name"]
-    assert (
-        service_inspect["Spec"]["TaskTemplate"]["ContainerSpec"]["Image"]
-        == service_spec["task_template"]["ContainerSpec"]["Image"]
-    )
-
```

Comment:
why was this removed?

Reply:
useless code, I removed the inspect_service code